drools-scenario-simulation/drools-scenario-simulation-backend/src/main/java/org/drools/scenariosimulation/backend/expression/DMNFeelExpressionEvaluator.java

```diff
@@ -89,8 +78,7 @@ protected FEEL newFeelEvaluator(AtomicReference<FEELEvent> errorHolder) {
 
     @Override
     protected Object internalLiteralEvaluation(String raw, String className) {
-        EvaluationContext evaluationContext = newEvaluationContext();
-        return executeAndVerifyErrors(feel -> feel.evaluate(raw, evaluationContext));
+        return executeAndVerifyErrors(feel -> feel.evaluate(raw));
```

Comment:
It should work with method ref now 🤔 
```suggestion
        return executeAndVerifyErrors(FEEL::evaluate);
```

Reply:
invalid suggestion ref https://github.com/kiegroup/drools/pull/3840#issuecomment-922996061